README.rst

```diff
@@ -101,7 +101,7 @@ the PyFluent-Visualization documentation.
 
 .. code:: python
 
-   from ansys.fluent.visualization.pyvista import Graphics
+   from ansys.fluent.visualization import Graphics
```

Comment:
- Import path looks better
- What we're calling `Graphics` here is a proxy / container for renderable solver objects. As such, we might consider a rename in the future.